src/views/v2/tickets/item/extra/available.php

```diff
@@ -24,9 +24,30 @@
 	return;
 }
 
+$available_count = $ticket->available();
+
+/**
+ * Allows hiding of "unlimited" to be toggled on/off conditionally.
+ *
+ * @since 4.11.1
+ *
+ * @var bool $show_unlimited    Whether to show the "unlimited" text.
+ * @var int  $available_count The quantity of Available tickets based on the Attendees number.
+ */
+$show_unlimited = apply_filters( 'tribe_tickets_block_show_unlimited_availability', true, $available_count );
+
+$context = [
+	'ticket'         => $ticket,
+	'show_unlimited' => (bool) $show_unlimited,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 2 space(s) between "'show_unlimited'" and double arrow, but found 1.

<sub>hash: 393547b8ff59d640d053512816dacf72</sub>